mne/viz/backends/_pysurfer_mayavi.py

```diff
@@ -43,7 +45,7 @@ def visible(self, state):
             self.pts.visible = state
 
 
-class _Renderer(object):
+class _Renderer(_BaseRenderer):
```

Comment:
Now that you have put all of these in... I wonder if we should make a `copy_base_doc_to_subclass_doc` decorator instead. It iterates over public methods of the base class and populates the doc of public methods of the subclasses. WDYT?

Reply:
That's the typical use-case for this class, I'll try to add this.

Reply:
Looks much cleaner!